### PR TITLE
Fix build errors: cspell, k8s-raw.gotmpl, etc

### PR DESCRIPTION
**Pull request by svatwork**
I fixed some of the build errors but I'm not sure how deep this rabbit hole goes. I fixed some errors, but I assume we agreed that the contributor is responsible for fixing the build so I was hoping @morriz could clean up the code. It's not that I don't want to help, but I also take in consideration that I might inadvertently break something else because I've not been working on this.

### CODE REVIEW

**Comment by Morriz** [comment]
@svatwork what should I clean up?

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ node_modules/
 .values/values-schema.yaml
 .env
 .secrets
+dist
```
[the remark this answers is not in the document]
Comment (svatwork) [reply .gitignore:15]: It's the CLI @Dunky13 is working on, I think it's also fine to omit it for now if you want.

**Comment by svatwork** [comment]
@Morriz I already did:

```
Generating k8s v1.19 manifests
638
helmfile-init.yaml: basePath=.
639
in helmfile.tpl/helmfile-init.yaml: failed to render values files "../values/k8s/k8s-raw.gotmpl": failed to render [../values/k8s/k8s-raw.gotmpl], because of template: stringTemplate:14:18: executing "stringTemplate" at <.Values.otomi>: map has no entry for key "Values"
640
[2021-07-13T 06:45:29.796] ERROR: Templates validation FAILED
```

But now there's a new error:

```
ERR  - raw/templates/resources.yaml: Duplicate 'ServiceAccount' resource 'default' in namespace 'drone-pipelines'
644
ERR  - raw/templates/resources.yaml: Duplicate 'ServiceAccount' resource 'default' in namespace 'otomi'
645
ERR  - raw/templates/resources.yaml: Duplicate 'ServiceAccount' resource 'default' in namespace 'maintenance'
646
[2021-07-13T 07:02:43.593] ERROR: Templates validation FAILED
647
The command '/bin/sh -c if [ "$SKIP_TESTS" = 'false' ]; then bin/ci-tests.sh; fi' returned a non-zero code: 1
648
+ logout_from_registry
649
+ echo -e '\n[Action Step] Log out from registry...'
650
+ docker logout ghcr.io
```

I don't know what else since the commit went to master last night without a successful build.

**Comment by svatwork** [comment]
Solved in https://github.com/redkubes/otomi-core/commit/121bf455eacc33055bf5d5c1aece15021dfe9bb0.

**Pull request closed without merging**